dm/dm/config/task.go

```diff
@@ -380,6 +386,20 @@ func (v *ValidatorConfig) Adjust() error {
 	if v.BatchQuerySize == 0 {
 		v.BatchQuerySize = DefaultValidatorBatchQuerySize
 	}
+	if v.MaxPendingRowSize == "" {
+		v.MaxPendingRowSize = DefaultValidatorMaxPendingRowSize
+	}
+
+	_, err := units.RAMInBytes(v.MaxPendingRowSize)
```

Comment:
how about use this func `ParseFileSize` in dm/pkg/dumpling/utils.go

Reply:
this method seems used to make `filesize` flag compatible with mydumper

from mydumper:
```
.. option:: --chunk-filesize -F

   Split tables into chunks of this output file size. This value is in MB
```
from dumpling
```
-F or --filesize
The approximate size of the output file. The unit should be explicitly provided (such as 128B, 64KiB, 32MiB, 1.5GiB)
```

but we don't have this burden for compatibility, will not change it now.